### ISSUE TITLE
Fix for "firewall" recommendation on installation page

### ISSUE THREAD

**Issue by sankarshanmukhopadhyay** [gluster/glusterdocs#402]
https://github.com/gluster/glusterdocs/blob/master/Install-Guide/Common_criteria.md needs to have a sensible recommendation for firewall topics

**Comment by nodesocket** [comment]
Agree. I was unable to actually find the ports to allow. The [good guide](https://www.howtoforge.com/tutorial/high-availability-storage-with-glusterfs-on-centos-7/) seems to recommend:

TCP ports: `111`, `24007`, `24008`, `24009-(24009 + number of bricks across all volumes)`
